l2-contracts/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "scripts": {
     "build": "hardhat compile",
-    "test:foundry": "forge test --zksync",
+    "test:foundry": "forge test --zksync --gas-limit 2000000000",
```

Comment:
Seems like you add config var for gas limit in the foundry.toml and here

Reply:
I removed from the config (this value has to be passed here)